### PR TITLE
feat: rename routes like login2

### PR DESCRIPTION
**Pull request by danielgent**
I always have to think which route is what as the names don't correspond. I don't mind what we call them, I'm just copying login2

### CODE REVIEW

**Review by markusahlstrand — COMMENTED** [review]
I would prefer using the paths from auth0 than copying anything from login2. Or am I missing something?

**Comment by danielgent** [comment]
> I would prefer using the paths from auth0 than copying anything from login2. Or am I missing something?

What paths are there on auth0 though? there's no codes on universal login right?

Having `/u/code` as the form where a user enters an email makes no sense :wink:

**Comment by markusahlstrand** [comment]
Now I understand this PR :) Totally agree

File: test/integration/__image_snapshots__/breakit-customizations-spec-ts-test-integration-breakit-customizations-spec-ts-only-allows-existing-breakit-users-to-progress-to-the-enter-code-step-2-snap.png
Comment (danielgent) [inline test/integration/__image_snapshots__/breakit-customizations-spec-ts-test-integration-breakit-customizations-spec-ts-only-allows-existing-breakit-users-to-progress-to-the-enter-code-step-2-snap.png:1]: this snapshot was testing that the _create account_ link isn't shown if signups are disabled

File: test/integration/breakit-customizations.spec.ts [hunk 1]
```diff
@@ -133,7 +135,7 @@ test("only allows existing breakit users to progress to the enter code step", as
   // if sign ups are disabled, the create account link should not be shown
   // ----------------------------
 
-  const loginFormNoSignupResponse = await oauthClient.u.login.$get({
+  const loginFormNoSignupResponse = await oauthClient.u["enter-code"].$get({
```
Comment (danielgent) [inline test/integration/breakit-customizations.spec.ts:138]: :eyes: this isn't correct. this should be `u["enter-password"]`, that's the whole point I did the PR I suppose! The previous naming was confusing

I'll revert this change in a new PR, and add an explicit assertion that _the create account link should not be shown_ (as per the comment above). It was lazy on my behalf to use a snapshot image only for this test

**Pull request merged**